packages/contracts/tasks/transfer-ownership.ts

```diff
@@ -0,0 +1,139 @@
+import '@nomiclabs/hardhat-ethers'
+import assert from 'assert'
+import readline from 'readline'
+
+import { task } from 'hardhat/config'
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+
+import { getDeploymentAddress } from '../src/deploy-utils'
+
+const isL1Network = (hre: HardhatRuntimeEnvironment): boolean => {
+  const networkName = hre.network.name
+  const l1Network = ['mainnet', 'sepolia', 'easel', 'devnetL1']
+  return l1Network.includes(networkName)
+}
+const transferProxyAdminOwnership = async (
+  hre: HardhatRuntimeEnvironment,
+  newOwner: string
+) => {
+  console.log('transfer ProxyAdmin ownership to TimeLock')
+  const proxyAdminAddress = isL1Network(hre)
+    ? await getDeploymentAddress(hre, 'ProxyAdmin')
+    : await getDeploymentAddress(hre, 'ProxyAdminProxy')
+  let proxyAdmin = await hre.ethers.getContractAt(
+    'ProxyAdmin',
+    proxyAdminAddress
+  )
+  const currentProxyAdminOwner = await proxyAdmin.owner()
+  if (currentProxyAdminOwner === newOwner) {
+    console.log(
+      'skip the ProxyAdmin owner transfer process because the owner has already been transferred.'
+    )
+    return
+  }
+
+  proxyAdmin = proxyAdmin.connect(
+    hre.ethers.provider.getSigner(currentProxyAdminOwner)
+  )
+
+  const tx = await proxyAdmin.transferOwnership(newOwner, {
+    from: currentProxyAdminOwner,
+  })
+  await tx.wait()
+
+  assert((await proxyAdmin.owner()) === newOwner)
+  console.log('successfully transferred ownership of ProxyAdmin')
+}
+
+const transferSecurityCouncilTokenOwnership = async (
+  hre: HardhatRuntimeEnvironment,
+  newOwner: string
+) => {
+  console.log(
+    'transfer SecurityCouncilToken ownership to SecurityCouncilTokenOwner'
+  )
+  const scTokenProxyAddress = await getDeploymentAddress(
+    hre,
+    'SecurityCouncilTokenProxy'
+  )
+  let scToken = await hre.ethers.getContractAt(
+    'SecurityCouncilToken',
+    scTokenProxyAddress
+  )
+  const currentScTokenOwner = await scToken.owner()
+  if (
+    hre.ethers.utils.getAddress(currentScTokenOwner) ===
+    hre.ethers.utils.getAddress(newOwner)
+  ) {
+    console.log(
+      'skip the SecurityCouncilToken owner transfer process because the owner has already been transferred.'
+    )
```

Comment:
As I mentioned in previous comment, the owner has been changed to same address. We need to return this function when `newOwner` address is same as `currentScTokenOwner`.
```suggestion
    )
    return
```